### ISSUE TITLE
Make sure to parse globals, things other than functions

### ISSUE THREAD

**Issue by vsoch** [buildsi/Smeagle#65]

**Comment by vsoch** [comment]
@hainest can we do this soon?

**Comment by hainest** [comment]
Yeah, we can work on this. I'm curious what the "things other than functions" would include beyond exported variables. I can't think of anything offhand.

For exported variables, we'll need to see how they are tagged in the ELF symbol tables. Dyninst can definitely give them to us, I'm just not sure where they are.

**Comment by vsoch** [comment]
I think @mplegendre has just mentioned globals, and I added the "things other than functions" in case there as something else. Maybe something will call sites? We can rename this issue to just globals if that's the only-other-thing!

**Comment by vsoch** [comment]
Something like `STB_GLOBAL=1`? https://stackoverflow.com/questions/17159666/where-are-global-variables-located-in-the-elf-file. Can Dyninst see that?
